feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-method.component.ts

```diff
@@ -90,16 +87,9 @@ export class CheckoutPaymentMethodComponent implements OnInit, OnDestroy {
       .pipe(
         filter((state) => !state.loading),
         map((state) => state.data),
-        tap((paymentInfo: any) => {
+        tap((paymentInfo) => {
           if (paymentInfo && !!Object.keys(paymentInfo).length) {
-            if (paymentInfo['hasError']) {
```

Comment:
dead code that came from year 1 of Spartacus, and wasn't ever removed

https://github.com/SAP/spartacus/pull/7247/files#diff-6a20f7cb50b2fa544425396960af44eae0a82887f4d4b057d114f36e81aa7f03R89

https://github.com/SAP/spartacus/pull/2189/files#diff-c45eb1dbd0ef5f712ba7188480c4a3d9d79f1b9917857727c4cbb1e9f603f3f5R77

This is already handled elsewhere in the `bad-request.handler.ts`  
https://github.com/SAP/spartacus/blob/develop/projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts#L73

![Screen Shot 2021-12-01 at 5 24 28 PM](https://user-images.githubusercontent.com/29099466/144323919-3a675f78-c650-4ae7-a90b-368f55245a7b.png)
![Screen Shot 2021-12-01 at 5 22 59 PM](https://user-images.githubusercontent.com/29099466/144323937-058562ce-a8a6-405b-af93-cdfc366fcf91.png)